project/MicroService.scala

```diff
@@ -27,7 +26,7 @@ trait MicroService {
   lazy val scoverageSettings = {
     Seq(
       ScoverageKeys.coverageExcludedPackages := "<empty>;app.*;config.*;testOnlyDoNotUseInAppConf.*;views.*;uk.gov.hmrc.*;prod.*;models.*;services.ERSRequest",
-      ScoverageKeys.coverageMinimum := 70,
+      ScoverageKeys.coverageMinimum := 87,
```

Comment:
Why 87% of all numbers?

Reply:
because tests are at just under 87%, my view is that the coverage min should be what it's at so if someone comes along makes a change and coverage drops it should be immediately known